src/locales/en.json

```diff
@@ -431,27 +431,53 @@
   "nft_book_hero_skip_animation_button": "Skip Animation",
   "nft_book_shelf_multiple_nft_class_dialog_title": "Please select a version",
   "nft_claim_access_nft_book": "Proceed",
+  "nft_claim_author": "author",
   "nft_claim_claim": "Proceed",
-  "nft_claim_claimed": "NFT is claimed",
-  "nft_claim_claimed_download": "Or just download the eBook?",
-  "nft_claim_claimed_gift_fromName": "—— {name}",
-  "nft_claim_claimed_gift_title": "This book was selected by {name} as a special gift for you, hope you like it!",
-  "nft_claim_claimed_gift_toName": "To {name}:",
+  "nft_claim_claim_header_title": "Claimed eBook",
+  "nft_claim_claimed_button_start_reading": "Start reading",
+  "nft_claim_claimed_button_view_collection": "View collection",
+  "nft_claim_claimed_content_autoDelivery": "{publisher} has just sent you {name}, Visit your dashboard to start reading.",
+  "nft_claim_claimed_content_manualDelivery": "Please wait for the author to sign and send the eBook, which may take about 1-3 days. Any update will be notified via email. Thank you for your support!",
+  "nft_claim_claimed_download": "Other Ways to Obtain the eBook",
   "nft_claim_claimed_nft_book": "You have successfully completed the required verification to claim your eBook. Please await the addition of the creator's digital signature or personalized message, after which the eBook will be delivered to you. Thank you for your valued support!",
-  "nft_claim_claimed_retry_button": "Try Again",
-  "nft_claim_claimed_view_button": "View Collected",
-  "nft_claim_claimed_view_class_button": "View Details",
-  "nft_claim_claiming": "Verifying",
+  "nft_claim_claimed_message_title": "{name}'s message:",
+  "nft_claim_claimed_title_autoDelivery": "You have successfully claimed your eBook",
+  "nft_claim_claimed_title_congratulations": "Congratulations",
+  "nft_claim_claimed_title_manualDelivery": "Your book is on its way!",
+  "nft_claim_claiming_content": "Verifying",
   "nft_claim_class_not_found": "NFT Class not found",
-  "nft_claim_enter_address_errorMessage": "Only LIKE wallets (like1...) are allowed",
-  "nft_claim_enter_address_label": "Enter your wallet address to add to your bookshelf",
-  "nft_claim_enter_address_placeholder": "like1.... or connect by logging in",
-  "nft_claim_error_message": "Error occurs when adding to bookshelf {error}",
+  "nft_claim_confirmation_button_confirm": "Confirm",
+  "nft_claim_confirmation_content": "We have created a unique decentralized identity for you. \nsubtitle 交代 this is your identity in the web 3 world.\nWhere you could trade and exchange assets with this identity",

```

Comment:
Those `\n` should not be rendered unless we apply `whitespace-pre-wrap` class

Reply:
[already applied](https://github.com/likecoin/liker-land/pull/1588/files#diff-3ad5d4692f894ca8fc663ec45fcc3531aa9cd771545dab84ca79d69b8170f4d6R18)